src/instrumentations/https/HttpInstrumentation.ts

```diff
@@ -4,6 +4,15 @@ import { HttpHooks } from './http';
 import { Instrumentor } from '../instrumentor';
 
 export default class LumigoHttpInstrumentation extends Instrumentor<HttpInstrumentation> {
+
+  private readonly ignoredHostnames: string[];
+
+  constructor(ignoredHostnames: string[]) {
```

Comment:
please add tests to httpInstrumentation.test.js :)

Reply:
The current tests of `httpInstrumentation.test.js` are effectively worthless IMO: they just check that the configs are generated in a certain way, not that it ACTUALLY WORKS THAT WAY :-)

The fact that this patch works is proven by the fact that the Itests do not break by generating spans for the traces upload.

Reply:
you are right, the "config" tests we got are checking the specific instrumentation configuration, since this is unit testing which testing the functions and not necessarily how the instrumentation works. For testing the instrumentation itself we got the wrapper unit tests. Also, in the existing httpInstrumentation.test.js we're initializing the LumigoHttpInstrumentation object, with your change we will need to add 1 more argument to the constructor, so in any case you need to update those tests.